crates/task-impls/src/consensus.rs

```diff
@@ -1417,10 +1416,7 @@ where
                 .sign_validating_or_commitment_proposal::<I>(&leaf.commit());
             // TODO: DA cert is sent as part of the proposal here, we should split this out so we don't have to wait for it.
             let proposal = QuorumProposal {
-                block_header: VIDBlockHeader {
-                    block_number: leaf.height,
-                    commitment: *block_commitment,
-                },
+                block_header: TYPES::BlockHeader::new(*payload_commitment, parent_header.clone()),
```

Comment:
Nit: probably ok for this function to get a reference to the parent header, to avoid the clone